backend/controllers/GooglePlacesAPI.ts

```diff
@@ -0,0 +1,64 @@
+import {AxiosError, AxiosResponse} from "axios";
+
+class GooglePlacesAPI {
+
+    constructor() {
+        require('dotenv').config();
+    }
+
+    getGooglePlacesSearch(data: string): Promise<AxiosResponse<JSON>> {
+        let dataURI: string = encodeURIComponent(data);
+        let axios = require('axios');
```

Comment:
I think you should be able to add this in the import statement up top by changing it to:
```javascript
import axios, {AxiosError, AxiosResponse} from "axios";
```